go.mod

```diff
@@ -4,6 +4,6 @@ go 1.16
 
 require (
 	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20210520203318-a78632de1e26
-	k8s.io/apimachinery v0.20.5
-	sigs.k8s.io/controller-runtime v0.6.2
+	k8s.io/apimachinery v0.20.5 // indirect
```

Comment:
@OlegSternbergIBM not related to your PR - but it seems that these two lines are not necessary.
can you please remove the lines that ends with `//indirect` from go.mod?
probably leftovers of previous versions..